Styles.css

```diff
@@ -0,0 +1,107 @@
+* {
+    box-sizing: border-box;
+}
+
+html {
+    font-size: 62.5%;
+}
+
+body {
+    font-size: 2rem;
+    color: #ffffff;
+    background-image: url("BG.jpeg");
+    /*https://images.unsplash.com/photo-1536559692556-79e8be88e8ab?w=2008*/
```

Comment:
Commented stuff used to hold working old code while you replace it new code should be removed once you've confirmed that the new code works.